docs/snapshotting/network-for-clones.md

```diff
@@ -142,6 +142,19 @@ sudo ip route add 192.168.0.3 via 10.0.0.2
 
 **Full connectivity to/from the clone should be present at this point.**
 
+If there was a connection before snapshot, we will also save the address
```

Comment:
The ARP cache is owned by the guest and resides in it's memory and gets implicitly saved with the memory snapshot, Firecracker does not know itself anything about such caches in guests.

Reply:
Thanks @sandreim  and @acatangiu, updated the documentation, PTAL